src/encoded/types/target.py

```diff
@@ -65,13 +67,17 @@ def organism(self, properties=None, return_uuid=False):
     def name(self):
         return self.__name__
 
-    @calculated_property(condition='organism', schema={
+    @calculated_property(schema={
         "title": "Title",
         "type": "string",
     })
-    def title(self, request, label, organism):
-        organism_props = request.embed(organism, '@@object')
-        return u'{} ({})'.format(label, organism_props['scientific_name'])
+    def title(self, request, label, organism, investigated_as):
+        if organism is None:
+            # ENCD-4250 investigated_as must be just ['synthetic tag']
+            source = investigated_as[0].capitalize()
```

Comment:
What's preventing investigated as being patched as e.g. `["transcription factor", "synthetic tag"]`?

Reply:
As mentioned above, a target should not have `["transcription factor", "synthetic tag"]`. A test is added for this patch.